champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/paladin/passives/LifeBonds.java

```diff
@@ -109,32 +105,26 @@ public void grassAura() {
         }
     }
 
-    private void spawnParticlesAboveAllies(Player player, int distance) {
+    private void spawnParticlesAboveAllies(Player player, double distance) {
         List<Player> allies = getAllAllies(player, distance);
         for (Player ally : allies) {
             spawnParticleAboveHead(ally);
         }
     }
 
-    private List<Player> getAllAllies(Player player, int distance) {
-        List<Player> allies = new ArrayList<>();
+    private List<Player> getAllAllies(Player player, double distance) {
+        List<Player> allies = UtilPlayer.getNearbyAllies(player, player.getLocation(), distance);
         allies.add(player);
```

Comment:
I dont think you need to add yourself to this list, so you could remove this function entirely and just use getNearbyAllies directly

Reply:
im pretty sure I use this to do stuff to the caster too so id have to rewrite other parts if I was to remove this

Reply:
Yeah you are right, it filters out the current player by default